src/client/components/Autocomplete/Autocomplete.scss

```diff
@@ -2,22 +2,50 @@
 @import 'src/client/style/partials';
 
 .autocomplete {
-  width: 100%;
-  .autocomplete__listbox {
-    font-family: $font-family-1;
-    width: 100%;
-    cursor: pointer;
-    box-shadow: $shadow-3;
-    background-color: $ui-accent-light;
-    > * {
-      padding: $spacing-xxs;
+  transition: all .1s ease-out;
+
+  .autocomplete-dropdown {
```

Comment:
nested css do not need prefix. use prefix if css is not nested (preferred in this case)